pygenesys/utils/db_creator.py

```diff
@@ -1834,6 +1836,34 @@ def add_tech_input_split(connector, region, tech, time_periods, comm_list, eff_l
 
     connector.commit()
     return
+
+
+def create_lifetime_process(connector):
+    """
+    Creates the table LifetimeProcess.
+
+    ****************************
+    This function is INCOMPLETE.
```

Comment:
It would be helpful for future collaborators, or just your own sanity, if there was an associated issue and a little more detail here about what is incomplete.

Reply:
The function writes the table but does not populate it, as other functions do.